fil-proofs-tooling/src/bin/benchy/porep.rs

```diff
@@ -335,6 +323,12 @@ pub fn run_porep_bench<Tree: 'static + MerkleTreeTrait>(
     test_resume: bool,
     use_synthetic: bool,
 ) -> anyhow::Result<()> {
+    let features = if use_synthetic {
```

Comment:
Not a big deal, but eventually it would be good to see us reduce the boolean flags/checks and instead just pass in the enabled features for a pass through here.

Reply:
I though about it, but then you'd need to change the whole CLI parsing.

Reply:
I've started working on it, but it's not trivial. Hence merging this one and going from there then.